### PR TITLE
Replace hand-maintained list of previous benchmarks with pkg:build

### PR DESCRIPTION
**Pull request by kevmoo**
MUCH easier to maintain

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @jakemac53 – I love build 😄

**Comment by kevmoo** [comment]
@szakarias @sigmundch ?

File: tool/builder.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class _BenchmarkBuilder implements Builder {
+  @override
+  Future build(BuildStep buildStep) async {
+    var data = <String, String>{};
+
+    await for (var item in buildStep
+        .findAssets(Glob('benchmark/**/*.pb.json'))
+        .where((id) =>
+            id.pathSegments.length > 2 &&
+            id.pathSegments[0] == 'benchmark' &&
```
Comment (szakarias) [inline tool/builder.dart]: Is this line needed?

Comment (kevmoo) [reply tool/builder.dart]: Strictly, no. But I think it helps folks understand what's going on...

**Pull request merged**